### PR TITLE
Max: Tweaks on ValidateMaxContents

### PR DESCRIPTION
**Pull request by moonyuet**
## Changelog Description
This PR provides enhancements on ValidateMaxContent as follow:
1. Rename `ValidateMaxContents` to `ValidateContainers`
2. Add related families which are required to pass the validation(All families except `Render` as the render instance is the one which only allows empty container)

## Additional info
n/a

## Testing notes:
1. Create all instances except render
2. Publish
3. If the OP data of the container inside the instance is empty, it wont pass the validation

### CODE REVIEW

**Comment by BigRoy** [comment]
Would `ValidateInstanceHasMembers` be better?

Also, note that "container" is usually used for loaded content and here we are talking about publish instances.

File: openpype/hosts/max/plugins/publish/validate_containers.py
Comment (BigRoy) [inline openpype/hosts/max/plugins/publish/validate_containers.py:1]: Filename might still need updating?

**Review by LiborBatek — CHANGES_REQUESTED** [review]
Gives proper ```Validate``` when empty Instance present in Publishing...
image

however not working for ```Pointcloud``` Instance and gives following error in that particular case (it probably validates well the content if not empty tho)
image

**Comment by moonyuet** [comment]
> however not working for `Pointcloud` Instance and gives following error in that particular case (it probably validates well the content if not empty tho) image

This is the bug from the point cloud extractor, not related to this PR.

**Review by LiborBatek — APPROVED** [review]
Features related to this PR works fine. Can be approved then.

**Pull request merged**